packages/react/src/components/Divider/Divider.styles.js

```diff
@@ -1,13 +1,17 @@
 import { css } from '@emotion/react';
+import { useCustomTheme } from '../../hooks/useCustomTheme';
 
-const styles = {
-  divider: css`
+const useDividerStyles = () => {
+  const { colors } = useCustomTheme();
+  const divider = css`
     height: 2px;
     margin: 0 8px 8px;
     border: 0;
     border-radius: 2px;
-    background-color: #e4e7ea;
-  `,
+    background-color: ${colors.secondary};
+  `;
+
+  return { divider };
```

Comment:
is it possible to send the styles directly without destructuring them all the time?

Reply:
Since we have to send multiple styles, it's better to send them as objects and access them easily. If we don't do that, we won't be able to send multiple styles from one hook. Even for the same component, we might have to create separate hooks for each style, which I don't think will be good at all.